app/dashboard/views.py

```diff
@@ -4229,6 +4230,95 @@ def hackathon_save_project(request):
         })
 
 
+@login_required
+def get_project(request, project_id):
+    profile = request.user.profile if request.user.is_authenticated and hasattr(request.user, 'profile') else None
+
+    project = HackathonProject.objects.filter(pk=project_id).nocache().first()
+    if not project:
+        raise Http404("The project doesnt exists.")
+
+    hackathon_obj = HackathonEventSerializer(project.hackathon).data,
+    params = {
+        'project': {
+            'name': project.name,
+            'id': project.id,
+            'summary': project.summary,
+            'status': project.status,
+            'winner': project.winner,
+            'looking_members': project.looking_members,
+            'work_url': project.work_url,
+            'logo_url': project.logo.url if project.logo else staticfiles_storage.url(
+                f'v2/images/projects/logos/${project.id}.png'),
+            'prize': {
+                'id': project.bounty.id,
+                'title': project.bounty.title,
+                'avatar': project.bounty.avatar_url,
+                'org': project.bounty.org_display_name,
+                'org_url': project.bounty.org_profile.url if project.bounty.org_profile else '#',
```

Comment:
why return the `'#'` instead of None or blank? Probably better to handle the interpretation of a null value on the frontend

Reply:
removed